e2e/__tests__/__snapshots__/circusDeclarationErrors.test.ts.snap

```diff
@@ -16,6 +16,7 @@ FAIL __tests__/asyncDefinition.test.js
       14 |   });
       15 | });
 
+      at eventHandler (../../packages/jest-circus/build/eventHandler.js:143:11)
```

Comment:
should be stripped out from stack trace

Reply:
I tried, it was painful 😀 We could copy the stack from `asyncError` I guess